lib/extract_i18n/adapters/ruby_adapter.rb

```diff
@@ -77,7 +78,9 @@ def on_str(node)
 
     def ask_and_continue(i18n_key:, i18n_string:, interpolate_arguments: {}, node:)
       change = ExtractI18n::SourceChange.new(
-        i18n_key: "#{@file_key}.#{i18n_key}",
+        # we want the key to be the string itself so that we can extract it later
+        i18n_key: "#{i18n_string}",
+        i18n_string: "#{i18n_string}",
         i18n_string: i18n_string,
```

Comment:
Do we want to delete the second version of this key?

Reply:
you betcha! good catch!